packages/astro/src/assets/fonts/vite-plugin-fonts.ts

```diff
@@ -78,7 +80,10 @@ export function fontsPlugin({ settings, sync, logger }: Options): Plugin {
 	// We don't need to take the trailing slash and build output configuration options
 	// into account because we only serve (dev) or write (build) static assets (equivalent
 	// to trailingSlash: never)
-	const baseUrl = removeTrailingForwardSlash(settings.config.base) + URL_PREFIX;
+	const assetsDir = prependForwardSlash(
+		appendForwardSlash(joinPaths(settings.config.build.assets, ASSETS_DIR)),
+	);
+	const baseUrl = joinPaths(settings.config.base, assetsDir);
```

Comment:
Why don't we remove the trailing slash anymore? Also, the comment must be updated. Line 82 still talks about trailing slash

Reply:
I've updated the comment thanks! We don't need to care about the trailing slash here anymore because this is handled by each `UrlResolver`, which use `joinPaths()` under the hood